firmware/application/ui_navigation.cpp

```diff
@@ -733,6 +733,12 @@ void addExternalItems(NavigationView& nav, app_location_t location, BtnGridView&
                                "see Mayhem wiki and copy apps\n"
                                "to " + apps_dir.string() + " folder of SD card.");
                        }});
+
+        Painter painter;
+        painter.fill_rectangle(
+        {2 * 8, 0, portapack::display.width() - 4 * 8, portapack::display.height()},
```

Comment:
I don't like that this text string is covering the title/status bar icons, and I doubt whether it will alleviate customer questions regarding the missing apps.  At some point the user will notice that read Notice message.

Reply:
Make sense. I think I’ll remove this

Reply:
removed it and move the Notice tile to front